app/assets/javascripts/admin/templates/pages.js.handlebars

```diff
@@ -0,0 +1,49 @@
+
+<div class='list'>
+  <div class='well'>
+  <ul class='nav nav-list'>
+    {{#each content}}
```

Comment:
This is using the deprecated syntax. You should do something like `{{#each page in content}}` or something similar